lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1034,8 +1034,8 @@ public function getDateDiffExpression($date1, $date2)
     /**
      * Returns the SQL to add the number of given seconds to a date.
      *
-     * @param string  $date
-     * @param integer $seconds
+     * @param string         $date
+     * @param integer|string $seconds
```

Comment:
All of these should be reverted: we shouldn't accept strings, and should be very strict about this due to SQL injection concerns